### PR TITLE
update network-policy by adding egress and ipBlock usage

### PR DESCRIPTION
**Pull request by Lion-Wei**
update network-policy by adding egress and ipBlock usage

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5473)

### CODE REVIEW

**Comment by zacharysarah** [comment]
@cmluciano :wave: Bumping for review.

**Comment by cmluciano** [comment]
@Lion-Wei Can you remove the egress specific parts from here and just document the CIDR bits. 

https://github.com/kubernetes/kubernetes.github.io/pull/5529 should cover egress parts

cc @caseydavenport

File: docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -62,6 +77,13 @@ __podSelector__: Each `NetworkPolicy` includes a `podSelector` which selects the
 
 __ingress__: Each `NetworkPolicy` includes a list of whitelist `ingress` rules.  Each rule allows traffic which matches both the `from` and `ports` sections. The example policy contains a single rule, which matches traffic on a single port, from either of two sources, the first specified via a `namespaceSelector` and the second specified via a `podSelector`.
 
+__egress__: Each `NetworkPolicy` includes a list of whitelist `egress` rules. Each rule allows traffic which matches both the `ports` and `to` sections. The example policy contains a single rule, which matches traffic on a single port, to either of two sources, the first specified via a `namespaceSelector` and the second specified via a `podSelector`.
+
+__ipBlock__: `ipBlock` describes a particular CIDR that is allowed to
```
Comment (caseydavenport) [inline docs/concepts/services-networking/network-policies.md]: This is a bit asymmetrical - we don't document the `podSelector` and `namespaceSelector` bits here, which are equivalent. 

Not sure if we care though?

**Comment by Lion-Wei** [comment]
@cmluciano okay, that might be better, already changed, thanks

**Pull request merged**